be/src/runtime/tablets_channel.h

```diff
@@ -106,7 +106,12 @@ class TabletsChannel : public RefCountedThreadSafe<TabletsChannel> {
     static std::atomic<uint64_t> _s_tablet_writer_count;
 
     struct Sender {
-        std::mutex lock;
+        bthread::Mutex lock;
+
+        std::set<int64_t> _receive_sliding_window;
+        std::set<int64_t> _success_sliding_window;
+
+        int64_t _last_sliding_packet_seq = -1;
```

Comment:
```suggestion
        std::set<int64_t> receive_sliding_window;
        std::set<int64_t> success_sliding_window;

        int64_t last_sliding_packet_seq = -1;
```

Reply:
According to our coding guide, the name of public variables should not start with an underscore.